sql/flow_config.go

```diff
@@ -44,7 +45,17 @@ func GetPypiVersion(projectURL string) (string, error) {
 		return "", fmt.Errorf("error parsing response for project version %w", err)
 	}
 
-	return resp.Info.Version, nil
+	versions := make([]*version.Version, len(resp.Releases))
+	index := 0
+	for release := range resp.Releases {
+		versions[index], err = version.NewVersion(release)
+		if err != nil {
+			return "", fmt.Errorf("error parsing release version %w", err)
```

Comment:
nitpick: should we continue processing the rest of the version and pick one from all the valid ones instead of erroring? IMO some consistent result is always better than no result

Reply:
Since we control this package and strictly use semver I think it should not be a problem, but simply skipping this version makes sense to me nevertheless.